openid-sharedsignals-framework-1_0.md

```diff
@@ -602,6 +602,18 @@ authorization_schemes
   security considerations, make the authorization_schemes attribute
   publicly accessible without prior authentication.
 
+default_subjects
+
+> OPTIONAL. A string indicating the default behavior of newly created streams. The value
+  MUST be either "ALL" or "NONE". If not provided, the default value is "NONE".
```

Comment:
Requiring a certain default behavior when the value is not present will make this backward incompatible. Can we say that if not present, "the Transmitter behavior in this regard is unspecified"